spec/ameba/rule/layout/trailing_blank_lines_spec.cr

```diff
@@ -4,35 +4,55 @@ module Ameba::Rule::Layout
   subject = TrailingBlankLines.new
 
   describe TrailingBlankLines do
-    it "passes if there is no blank lines at the end" do
-      source = Source.new "no-blankline"
+    it "passes if there is a blank line at the end of a source" do
+      source = Source.new "a = 1\n ", normalize: false
```

Comment:
I think the last line should consist of only the line ending `\n`, without any whitespace.

Reply:
@Sija In fact, it is. I have updated this spec.